releasenotes/notes/add-trace-injector-for-botocore-eventbridge-7dd2e56134b9472f.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Add the ability to inject trace context into AWS EventBridge events in the botocore Patcher.
```

Comment:
Also kinesis and sns? Probably also want to use the "distributed tracing" wording here for consistency.

Should also mention how to enable/disable the feature

Reply:
Thanks, updated the release note